src/sidebar/store/modules/direct-linked.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const util = require('../util');
+
+function init(settings) {
+  return {
+    /**
+     * The id of the direct-linked group.
+     * @type {string}
+     */
+    directLinkedGroupId: settings.group || null,
+
+    /**
+     * The id of the direct-linked annotation's group.
+     * @type {string}
+     */
+    directLinkedAnnotationId: settings.annotations || null,
+
+    /**
+     * Indicates that an error occured in retrieving/showing the direct linked group.
+     * This could be because:
+     * - the group does not exist
+     * - the user does not have permission
+     * - the group is out of scope for the given page
+     * @type {boolean}
+     */
+    directLinkedGroupFetchFailed: false,
+  };
+}
+
+const update = {
+  UPDATE_DIRECT_LINKED_GROUP_FETCH_FAILED(state, action) {
+    return {
+      directLinkedGroupFetchFailed: action.directLinkedGroupFetchFailed,
+    };
+  },
+  UPDATE_DIRECT_LINKED_GROUP_ID(state, action) {
+    return {
+      directLinkedGroupId: action.directLinkedGroupId,
+    };
+  },
+  UPDATE_DIRECT_LINKED_ANNOTATION_ID(state, action) {
+    return {
+      directLinkedAnnotationId: action.directLinkedAnnotationId,
+    };
+  },
+  CLEAR_DIRECT_LINKED_IDS() {
+    return {
+      directLinkedAnnotationId: null,
+      directLinkedGroupId: null,
+    };
+  },
+};
+
+const actions = util.actionTypes(update);
+
+/**
+ * Set the direct linked group id.
+ */
+function setDirectLinkedGroupId(groupId) {
+  return {
+    type: actions.UPDATE_DIRECT_LINKED_GROUP_ID,
+    directLinkedGroupId: groupId,
+  };
+}
```

Comment:
I'm going to leave these two actions here for now as they are used by sidebar content tests. However as a design principle, I should note that it isn't necessary to have "setters" for all the state in the store like you might have for objects in OO design. Idiomatically actions should be "things that happened" in the program (for example, data was fetched, the selection was cleared, the active document or tab changed).